### ISSUE TITLE
What's the license?

### ISSUE THREAD

**Issue by zougloub** [yuyuaqwq/atomkv#3]
I took a look at the repo after hearing of it via lmdb's hyc, and I couldn't find any information regarding the license; such information would be useful for users of the lib.
Can you clarify?

Thank you,

**Comment by yuyuaqwq** [comment]
Of course, I should add license information. The reason I haven't added it is that yudb is not yet mature enough, and I don't have enough understanding of license-related information. However, upon your reminder, I tried to learn more about it and have added the MIT license to it. Thank you for bringing this to my attention!
